Feature: Use the current working directory (of the opened buffer) to execute the git commands (#51)
stsewd/fzf-checkout.vim: doc/fzf-checkout.txt

- `execute`: A string to be executed when performing this action.
  You can make use of the following placeholders:
  - `{git}`: Replaced with the value from |g:fzf_checkout_git_bin|
  - `{wd}`: Replaced, if |g:fzf_checkout_use_current_buf_cwd| is enabled, with

>     the correct working directory. This option is manadged automatically by
>     the plugin.

Reviewer: ```suggestion
    the directory from the current buffer or the current working directory.
```

>
   let g:fzf_checkout_merge_settings= v:true
<
                                          *g:fzf_checkout_use_current_buf_cwd*
g:fzf_checkout_use_current_buf_cwd~

> Set the |g:fzf_checkout_use_current_buf_cwd| variable to operate in the
> correct directory for the current buffer. This will execute the `git` commands
> in the folder containing the file displayed in the current buffer. This
> facilitate the work in a multi-git-repository project (with submodules,...).
> This option is disable by default.

Reviewer: ```suggestion
If true, set the `{cwd}` placeholder to the directory from the current buffer.
Useful when working in a multi-git-repository project (with submodules for example).
This option is disable by default.
```

>

>     let g:fzf_checkout_use_current_buf_cwd = v:true

Reviewer: I'm using the codeblocks to show the defaults

```suggestion
    let g:fzf_checkout_use_current_buf_cwd = v:false
```
